crates/teloxide-core/CHANGELOG.md

```diff
@@ -21,6 +21,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
```

Comment:
And actually your described changes should be here, and links to the PRs will be grouped together then (the previous 7.1 and yours)